nexus/social_media/services.py

```diff
@@ -0,0 +1,36 @@
+# Third Party Stuff
+import facebook
+from django.conf import settings
+
+# nexus Stuff
+from nexus.base import exceptions
+# Nexus Stuff
+from nexus.social_media.models import Post
+
+
+def get_fb_page_graph():
+    graph = facebook.GraphAPI(settings.FB_USER_ACCESS_TOKEN)
+    accounts = graph.get_object('me/accounts')['data']
+    page_access_token = None
+    for account in accounts:
+        if account['id'] == settings.FB_PAGE_ID:
+            page_access_token = account['access_token']
+            break
+    if page_access_token is None:
+        raise exceptions.WrongArguments("Facebook Page access token could not be found")
+    page_graph = facebook.GraphAPI(page_access_token)
+    return page_graph
+
+
+def post_to_facebook(post_id):
+    post = Post.objects.get(pk=post_id)
+    page_graph = get_fb_page_graph()
+    if post.image:
```

Comment:
why not just do `if post.image and post.text`?

Reply:
I wanted to break the checks in two parts. If it contains image or not. This line ensures that the post contains image the nested if checks if there's also text associated with it.